backend/core/views.py

```
@@ -1102,6 +1196,14 @@
     def effort(self, request):
         return Response(dict(AppliedControl.EFFORT))
 
+    @action(detail=False, name="Get all applied controls owners")
+    def owner(self, request):
+        return Response(
+            UserReadSerializer(
```

Ruff (0.8.2)

`UserReadSerializer` may be undefined, or defined from star imports

(F405)